openid-federation-1_0.xml

```diff
@@ -3447,7 +3468,7 @@
 		of the Trust Anchor's Entity Configuration,
                 verify that the Trust Mark contains
                 a <spanx style="verb">delegation</spanx>
-                claim.
+                claim.  (TBD WHAT TO DO IF IT DOESN'T?)
```

Comment:
Regarding delegation, this might be common sense, but I don't see any instruction for validating that the `trust_mark_id` claim from the Trust Mark JWS is the same as the `trust_mark_id` in Trust Mark Delegation JWS, and that the `iss` from the Trust Mark JWS is the same as the `sub` claim in Trust Mark Delegation JWS. Without this check, Trust Mark Issuer can include `delegation` JWS that is verifiable using `jwks` and `sub` claims from  `trust_mark_owners`, but which actually doesn't say that the Issuer is "allowed" to issue that particular Trust Mark.